pkg/util/xc/dummy.go

```diff
@@ -0,0 +1,10 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package xc
+
+func doNothing() int64 {
+	return 0
+}
```

Comment:
probably a temp file you can remove now :)

Reply:
No.  go vet complains that pkg/util/xc doesn't contain any files otherwise.